src/manifest/manifest-create.action.ts

```diff
@@ -2,7 +2,6 @@ import { ManifestCreateOptions } from "./manifest-create.options";
 import { ManifestDto, ManifestVersions } from "./dto";
 import { readFile, writeFile } from "fs/promises";
 import { checkFileOverwrite } from "../lib/file";
-import fetch from "node-fetch";
```

Comment:
If it's not used anywhere you can drop it from package.json as well.

Also, that's a breaking change so it would be good if you include this information in the commit message so that semantic release can pick this up.